test/15_target-hostapp.ts

```diff
@@ -289,7 +297,9 @@ export default () => {
 								const [service] = serviceInstalls[0].installs__service;
 								expect(service).to.have.property(
 									'id',
-									fx.services['intel-nuc_service1'].id,
+									fx.services[
+										!isEsr ? 'intel-nuc_service1' : 'intel-nuc-esr_service1'
```

Comment:
```suggestion
										isEsr ? 'intel-nuc-esr_service1' : 'intel-nuc_service1'
```
Same as above, easier to read/parse this way